### ISSUE TITLE
Crashing during export on 1.0.2

### ISSUE THREAD

**Issue by nickrenda** [monarchmoney/mint-export-extension#15]
I ran the extension on 1.0.2 unpacked in developer mode, and cannot export my balances without the extension crashing. I also get an error from mint.com until I close and re-open chrome saying mint.com is unavailable. Wonder if mint is rate limiting? I can usually get to about 40 accounts before it crashes. 

I have 205 total accounts

<img width="689" alt="Screenshot 2023-11-14 at 5 02 48 PM" src="https://github.com/monarchmoney/mint-export-extension/assets/19145350/9e37cfdf-7b97-4f9d-9c65-6dbfa68941df">

**Comment by nickrenda** [comment]
Copying the errors I saw here from chrome developer mode:

Error 1

Access to fetch at 'https://mint.intuit.com/pfm/v1/user' from origin 'chrome-extension://jekpaapjbcmkbilkckepljakpndlpepp' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
Context
src/pages/popup/index.html
Stack Trace
src/pages/popup/index.html:0 (anonymous function)
Nothing to see here, move along.

Error 2:

Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
Context
src/pages/popup/index.html
Stack Trace
src/pages/popup/index.html:0 (anonymous function)
Nothing to see here, move along.

**Comment by vanessa** [comment]
Thanks for reporting this. Can you test if this error still happens with https://github.com/monarchmoney/mint-export-extension/releases/tag/1.0.3?

**Comment by nickrenda** [comment]
issue persists on 1.0.3. Got to about 40 accounts before the extension crashed. reloading the page shows me: 

<img width="910" alt="Screenshot 2023-11-14 at 7 52 05 PM" src="https://github.com/monarchmoney/mint-export-extension/assets/19145350/c4b43101-3c79-4c29-a37b-4ffca8cb9a7b">

**Comment by vanessa** [comment]
It seems that the issue may be related to rate limiting, as you mentioned. Could you try changing the value of `MINT_RATE_LIMIT_DELAY_MS` locally and test to see if it resolves the problem?

**Comment by nickrenda** [comment]
I'm actually not sure how to do that - I'm just savvy enough to download and post to GitHub, but im not sure where or how to change this value. If you can direct me to what file it's in I can try and edit it? Do I need to edit the value and then manually rebuild the package myself?

**Comment by devinellis** [comment]
1.0.5 looks to have fixed the issue, I was able to download balance history for 52 accounts.

**Comment by nickrenda** [comment]
Yeah 1.0.5 also just fixed it for me - noticed the rate was much slower too. Only other item (maybe this is already a feature request) is that while it reported some accounts could not download, I have no way of knowing which ones these were. I presume these were probably duplicate or hidden accounts with no data to download, but it would be great if the app could report out a list of what accounts these were so I could confirm. 


<img width="351" alt="((**Accounts" src="https://github.com/monarchmoney/mint-export-extension/assets/19145350/69cccf3a-e294-4082-a8fb-cd5dc476f2bd">

**Comment by mschmidty4** [comment]
I found the failed downloads were directly tied to having the "Hide Accounts" set to something other than "Show Everywhere". Changed my accounts to "Show Everywhere" and no more failures!
![image](https://github.com/monarchmoney/mint-export-extension/assets/62256174/34929747-8e89-46fb-9738-bfe51604439b)

**Comment by davidmtml** [comment]
I'm still having this problem, I have 210 accounts.

**Comment by mschmidty4** [comment]
> I'm still having this problem, I have 210 accounts.

Did you try switching them to "show everywhere" in the "hide accounts" tab?

**Comment by davidmtml** [comment]
> > I'm still having this problem, I have 210 accounts.
> 
> Did you try switching them to "show everywhere" in the "hide accounts" tab?

Yep, all of my accounts are set to "show everywhere".  Just tried it again, this time it crashed after about 40 accounts.

**Comment by davidmtml** [comment]
> [mschmidty4](/mschmidty4)

Just to clarify.  My issue is the original issue listed in this report, where you get the "Oops, we ran into an error" webpage, and can't log back into Mint for 10 minutes.  Not the "xx accounts failed to download" error.
